### ISSUE TITLE
Minimap disappears when closing Peek Definition window

### ISSUE THREAD

**Issue by TanelS** [Microsoft/vscode#28449]
- VSCode Version: 1.13.0
- OS Version: Win 10 Pro

Steps to Reproduce:

1. Open "Peek Definition";
2. Close the window by clicking a cross on the upper-right corner;
3. Minimap disappears.

Minimap reappears only after reloading the project. Closing the peek-window by pressing Esc does not close the minimap at first, it remains visible. But as you right-click on something the minimap disappears.

**Comment by ramya-rao-a** [comment]
Duplicate of #28093

The fix is available in the Insiders.

**Comment by ramya-rao-a** [comment]
cc @alexandrudima @rebornix

**Comment by mxrtos** [comment]
Minimap now disappears when running "Go Back" (workbench.action.navigateBack) on code-insiders_1.14.0-1497948192_amd64.deb
